ui/client/components/ViewFeatures.js

```diff
@@ -364,7 +364,6 @@ const ViewFeatures = withStyles(() => ({
           LoadingOverlay: CustomLoadingOverlay
         }}
         loading={featuresLoading}
-        getRowId={(row) => `${row.owner_dataset.id}-${row.name}`}
```

Comment:
This was an old addition, not necessary anymore as the features include an id, which DataGrid uses by default. The existing feature IDs are exactly this (a concatenation of their owner dataset and their name).